pkgs/top-level/all-packages.nix

```diff
@@ -691,10 +691,20 @@ with pkgs;
 
   broadlink-cli = callPackage ../tools/misc/broadlink-cli {};
 
-  fetchpatch = callPackage ../build-support/fetchpatch { }
-    // {
-      tests = pkgs.tests.fetchpatch;
-    };
+  fetchpatch = callPackage ../build-support/fetchpatch {
+    # 0.3.4 would change hashes: https://github.com/NixOS/nixpkgs/issues/25154
+    patchutils = buildPackages.patchutils_0_3_3;
+  } // {
+    tests = pkgs.tests.fetchpatch;
+    version = 1;
```

Comment:
Since fetchurl now support pname+version we could set this via that but not super important.

Reply:
That'd be on the result of fetchpatch, whereas the test needs it to have a version for the function itself.
I think having the version on the result (derivation) would be confusing. (Why does my patch have a version I didn't specify?)